2021-10-22 10:59:33,784-->get_element.py get_element 20-->ERROR No section: 'element'
2021-10-22 15:20:51,920-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:22:07,782-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:23:02,239-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:28:04,946-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:30:43,194-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:31:12,761-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:38:32,365-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:38:57,829-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:40:25,546-->driver.py open_driver 39-->INFO 浏览器启动成功
2021-10-22 15:40:25,820-->login.py login_page 20-->INFO 登录成功
2021-10-22 15:46:20,827-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 15:47:59,961-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 15:54:56,743-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 15:56:16,699-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 15:56:17,018-->login.py choose_space 31-->ERROR Message: invalid selector: Unable to locate an element with the xpath expression #app > div > div > main > main > div.power_manage > div > div > div > div > div > div > div > table > tbody > tr > td:nth-child(1) > span because of the following error:
SyntaxError: Failed to execute 'evaluate' on 'Document': The string '#app > div > div > main > main > div.power_manage > div > div > div > div > div > div > div > table > tbody > tr > td:nth-child(1) > span' is not a valid XPath expression.
  (Session info: chrome=94.0.4606.61)

2021-10-22 15:58:23,610-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 15:58:23,928-->login.py login_page 20-->INFO 登录成功
2021-10-22 16:01:13,454-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 16:02:08,466-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 16:02:08,683-->login.py login_page 20-->INFO 登录成功
2021-10-22 16:08:47,489-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 16:08:47,702-->login.py login_page 20-->INFO 登录成功
2021-10-22 16:09:04,662-->choose_space.py choose_space 18-->INFO 进入空间测试复制FS空间
2021-10-22 16:14:46,669-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 16:14:46,866-->login.py login_page 20-->INFO 登录成功
2021-10-22 16:14:46,874-->choose_space.py choose_space 21-->ERROR Message: no such element: Unable to locate element: {"method":"css selector","selector":"#app > div > div > main > main > div.power_manage > div > div > div > div > div > div > div > table > tbody > tr > td:nth-child(1) > span"}
  (Session info: chrome=94.0.4606.61)

2021-10-22 16:16:37,457-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 16:16:37,735-->login.py login_page 20-->INFO 登录成功
2021-10-22 16:16:37,750-->choose_space.py choose_space 21-->ERROR Message: no such element: Unable to locate element: {"method":"css selector","selector":"#app > div > div > main > main > div.power_manage > div > div > div > div > div > div > div > table > tbody > tr > td:nth-child(1) > span"}
  (Session info: chrome=94.0.4606.61)

2021-10-22 16:20:58,184-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 16:20:58,407-->login.py login_page 20-->INFO 登录成功
2021-10-22 16:20:58,417-->choose_space.py choose_space 21-->ERROR Message: no such element: Unable to locate element: {"method":"css selector","selector":"#app > div > div > main > main > div.power_manage > div > div > div > div > div > div > div > table > tbody > tr > td:nth-child(1) > span"}
  (Session info: chrome=94.0.4606.61)

2021-10-22 16:24:02,571-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 16:24:02,785-->login.py login_page 20-->INFO 登录成功
2021-10-22 16:24:51,521-->choose_space.py choose_space 22-->ERROR 'WebDriver' object has no attribute 'implicitywait'
2021-10-22 19:55:13,710-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 19:55:14,321-->login.py login_page 20-->INFO 登录成功
2021-10-22 19:55:14,556-->choose_space.py choose_space 19-->INFO 进入空间:测试复制FS空间
2021-10-22 19:56:08,907-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 19:56:09,563-->login.py login_page 20-->INFO 登录成功
2021-10-22 19:56:27,163-->choose_space.py choose_space 19-->INFO 进入空间:测试复制FS空间
2021-10-22 20:22:15,841-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:22:16,281-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:23:20,623-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:23:21,068-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:25:07,583-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:25:08,075-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:26:26,085-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:26:26,578-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:30:04,705-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:30:05,273-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:31:00,396-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:31:00,994-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:36:56,269-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:36:56,705-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:41:51,487-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:41:52,082-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:42:28,677-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:42:29,395-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:46:00,665-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:46:01,296-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:47:48,397-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:47:49,074-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:57:01,931-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:57:02,550-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:57:02,803-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-22 20:58:32,922-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-22 20:58:33,833-->login.py login_page 20-->INFO 登录成功
2021-10-22 20:58:34,124-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 10:39:38,387-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 10:39:38,583-->login.py login_page 20-->INFO 登录成功
2021-10-25 10:39:38,758-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 10:46:12,948-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 10:46:13,299-->login.py login_page 20-->INFO 登录成功
2021-10-25 10:46:13,504-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 10:47:44,855-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 10:47:45,222-->login.py login_page 20-->INFO 登录成功
2021-10-25 10:47:45,621-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 14:32:59,934-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 14:33:00,153-->login.py login_page 20-->INFO 登录成功
2021-10-25 14:33:00,354-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 15:04:41,784-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 15:04:42,028-->login.py login_page 20-->INFO 登录成功
2021-10-25 15:04:42,418-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 15:05:35,598-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 15:05:35,910-->login.py login_page 20-->INFO 登录成功
2021-10-25 15:05:36,874-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 17:30:50,463-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 17:30:50,955-->login.py login_page 20-->INFO 登录成功
2021-10-25 17:30:51,218-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 17:32:21,098-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 17:32:21,559-->login.py login_page 20-->INFO 登录成功
2021-10-25 17:32:21,799-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 17:34:47,953-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 17:34:48,479-->login.py login_page 20-->INFO 登录成功
2021-10-25 17:34:48,734-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 17:44:08,887-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 17:44:09,415-->login.py login_page 20-->INFO 登录成功
2021-10-25 17:44:09,730-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 17:44:11,003-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 20:52:33,847-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 20:52:34,290-->login.py login_page 20-->INFO 登录成功
2021-10-25 20:52:34,547-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 20:52:36,356-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:01:15,802-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:01:16,149-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:01:16,450-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:01:17,709-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:04:28,652-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:04:29,195-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:04:29,412-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:04:30,860-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:06:20,111-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:06:20,423-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:06:20,628-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:06:22,010-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:07:10,431-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:07:10,862-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:07:11,122-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:07:12,516-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:07:47,930-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:07:48,352-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:07:48,624-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:07:50,094-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:11:06,775-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:11:06,982-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:11:07,153-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:11:07,775-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:14:39,330-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:14:39,537-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:14:39,713-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:14:40,360-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:17:14,450-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:17:14,643-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:17:14,891-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:17:15,565-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:19:38,794-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:19:39,003-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:19:39,253-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:19:39,921-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:22:38,517-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:22:38,715-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:22:38,957-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:22:39,663-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:24:27,959-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:24:28,348-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:24:28,617-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:24:30,081-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:27:53,258-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:27:53,653-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:27:53,981-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:27:55,336-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:28:48,975-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:28:49,252-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:28:49,453-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:28:50,341-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:31:21,553-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:31:22,075-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:31:22,386-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:31:24,140-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:34:07,310-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:34:07,691-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:34:07,950-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:34:09,246-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:35:07,944-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:35:08,295-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:35:08,530-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:35:09,683-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:35:37,124-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:35:37,407-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:35:37,678-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:35:38,567-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:39:14,619-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:39:15,021-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:39:15,282-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:39:16,632-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:40:11,047-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:40:11,330-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:40:11,539-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:40:12,443-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 21:41:47,719-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 21:41:48,135-->login.py login_page 20-->INFO 登录成功
2021-10-25 21:41:48,387-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 21:41:49,784-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 22:10:42,614-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 22:10:43,074-->login.py login_page 20-->INFO 登录成功
2021-10-25 22:10:43,360-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 22:10:44,665-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 22:12:39,079-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 22:12:39,626-->login.py login_page 20-->INFO 登录成功
2021-10-25 22:12:39,994-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 22:12:41,327-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 22:15:57,781-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 22:15:58,232-->login.py login_page 20-->INFO 登录成功
2021-10-25 22:15:58,548-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 22:15:59,803-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 22:17:03,261-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 22:17:03,775-->login.py login_page 20-->INFO 登录成功
2021-10-25 22:17:04,253-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 22:17:05,737-->login.py base_message 40-->INFO 进入客户中心
2021-10-25 22:17:34,088-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-25 22:17:34,622-->login.py login_page 20-->INFO 登录成功
2021-10-25 22:17:34,998-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-25 22:17:36,303-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 11:02:27,767-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 11:02:28,356-->login.py login_page 20-->INFO 登录成功
2021-10-27 11:02:28,693-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 11:02:30,774-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 14:42:39,282-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 14:42:39,720-->login.py login_page 20-->INFO 登录成功
2021-10-27 14:42:40,126-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 14:42:41,604-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 14:44:29,960-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 14:44:30,401-->login.py login_page 20-->INFO 登录成功
2021-10-27 14:44:30,676-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 14:44:32,162-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 14:44:32,747-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 14:46:07,521-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 14:46:07,952-->login.py login_page 20-->INFO 登录成功
2021-10-27 14:46:08,472-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 14:46:10,086-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 14:46:10,652-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 14:46:12,673-->import_case.py upload_file 32-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/div/div/div/span[2]"}
  (Session info: chrome=94.0.4606.61)

2021-10-27 14:52:19,876-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 14:52:20,318-->login.py login_page 20-->INFO 登录成功
2021-10-27 14:52:20,695-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 14:52:22,287-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 14:52:22,863-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 14:56:00,109-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 14:56:00,671-->login.py login_page 20-->INFO 登录成功
2021-10-27 14:56:00,957-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 14:56:02,497-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 14:56:03,256-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:00:10,705-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:00:11,177-->login.py login_page 20-->INFO 登录成功
2021-10-27 15:00:11,501-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 15:00:13,034-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 15:00:13,693-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:02:11,739-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:02:12,177-->login.py login_page 20-->INFO 登录成功
2021-10-27 15:02:12,505-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 15:02:14,045-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 15:02:14,736-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:05:58,624-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:06:24,191-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:06:24,791-->login.py login_page 20-->INFO 登录成功
2021-10-27 15:06:25,375-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 15:06:27,412-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 15:06:28,230-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:06:28,459-->import_case.py upload_file 28-->INFO <selenium.webdriver.remote.webelement.WebElement (session="d8e3d4b2e7b391692859fa48caf1c90a", element="82a65d9f-b126-43d7-bf07-78f8b7336647")>
2021-10-27 15:08:22,359-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:08:22,995-->login.py login_page 20-->INFO 登录成功
2021-10-27 15:08:23,447-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 15:08:25,830-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 15:08:26,682-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:08:26,811-->import_case.py upload_file 28-->INFO 本次上传成功共:1条
2021-10-27 15:35:00,845-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:35:01,554-->login.py login_page 20-->INFO 登录成功
2021-10-27 15:35:01,857-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 15:35:03,747-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 15:35:04,824-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:35:04,860-->import_case.py upload_file 28-->INFO 本次上传成功共:1条
2021-10-27 15:51:54,057-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:51:54,522-->login.py login_page 20-->INFO 登录成功
2021-10-27 15:51:54,837-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 15:51:56,590-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 15:51:57,231-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:51:57,329-->import_case.py upload_file 28-->INFO 本次上传成功共:1条
2021-10-27 15:54:46,939-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 15:54:47,379-->login.py login_page 20-->INFO 登录成功
2021-10-27 15:54:47,765-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 15:54:49,477-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 15:54:50,121-->import_case.py upload_file 25-->INFO 导入数据
2021-10-27 15:54:50,569-->import_case.py upload_file 28-->INFO 本次上传成功共:1条
2021-10-27 16:12:14,081-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 16:12:14,521-->login.py login_page 20-->INFO 登录成功
2021-10-27 16:12:15,372-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 16:12:16,959-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 16:12:17,281-->modify_excle.py modify_info 35-->INFO 客户ID为：256401
2021-10-27 16:12:17,601-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 16:12:17,675-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 16:12:17,681-->login_out.py out_login 19-->ERROR 获取个人信息元素失败
2021-10-27 17:09:19,631-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:10:09,767-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:10:10,121-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:10:10,352-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:10:11,506-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:11:55,108-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:11:55,451-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:11:55,806-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:11:56,976-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:11:56,978-->login_out.py out_login 19-->ERROR 获取个人信息元素失败
2021-10-27 17:21:12,172-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:21:12,500-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:21:12,709-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:21:13,963-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:21:13,965-->login_out.py out_login 19-->ERROR 获取个人信息元素失败
2021-10-27 17:26:24,597-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:26:25,059-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:26:25,344-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:26:26,714-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:26:26,717-->login_out.py out_login 18-->ERROR e
2021-10-27 17:28:06,662-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:28:07,576-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:28:08,074-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:28:09,804-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:33:03,128-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:33:03,524-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:33:03,830-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:33:05,220-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:33:05,222-->login_out.py out_login 18-->ERROR e
2021-10-27 17:33:41,534-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:33:41,968-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:33:42,230-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:33:43,704-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:37:09,609-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:37:10,028-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:37:10,292-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:37:11,649-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:37:11,940-->login_out.py out_login 18-->ERROR e
2021-10-27 17:40:12,688-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:40:13,032-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:40:13,517-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:40:14,892-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:40:15,058-->login_out.py out_login 18-->ERROR e
2021-10-27 17:43:18,575-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:43:18,933-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:43:19,251-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:43:20,373-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:43:20,601-->login_out.py out_login 18-->ERROR e
2021-10-27 17:46:35,245-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:46:35,684-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:46:36,049-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:46:37,511-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:46:37,713-->login_out.py out_login 18-->ERROR e
2021-10-27 17:48:49,391-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:48:49,805-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:48:50,065-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:48:51,548-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:48:51,822-->login_out.py out_login 18-->ERROR e
2021-10-27 17:51:42,555-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:51:42,907-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:51:43,179-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:51:44,490-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:51:47,681-->login_out.py out_login 18-->ERROR e
2021-10-27 17:53:05,913-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:53:06,276-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:53:06,700-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:53:07,988-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:53:11,194-->login_out.py out_login 21-->ERROR e
2021-10-27 17:55:47,981-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:55:48,329-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:55:48,558-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:55:49,721-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:57:36,605-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:57:37,056-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:57:37,348-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:57:38,789-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:59:26,066-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 17:59:26,533-->login.py login_page 20-->INFO 登录成功
2021-10-27 17:59:26,859-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 17:59:28,263-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 17:59:31,656-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-27 18:01:15,704-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 18:01:16,152-->login.py login_page 20-->INFO 登录成功
2021-10-27 18:01:16,519-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 18:01:17,909-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 18:01:18,253-->modify_excle.py modify_info 35-->INFO 客户ID为：093641
2021-10-27 18:01:18,488-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 18:01:21,643-->login_out.py out_login 21-->ERROR e
2021-10-27 18:02:28,945-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 18:02:29,383-->login.py login_page 20-->INFO 登录成功
2021-10-27 18:02:29,640-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 18:02:31,326-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 18:02:31,579-->modify_excle.py modify_info 35-->INFO 客户ID为：290738
2021-10-27 18:02:31,914-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 18:02:32,017-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 18:02:37,290-->login_out.py out_login 21-->ERROR e
2021-10-27 19:03:32,900-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:03:33,343-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:03:33,624-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:03:35,213-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:03:35,530-->modify_excle.py modify_info 35-->INFO 客户ID为：043621
2021-10-27 19:03:35,916-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:03:35,954-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:03:41,203-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:07:29,959-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:07:30,399-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:07:30,827-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:07:32,356-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:07:32,672-->modify_excle.py modify_info 35-->INFO 客户ID为：807135
2021-10-27 19:07:33,031-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:07:33,074-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:09:19,801-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:11:25,707-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:11:26,557-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:11:29,485-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:11:30,846-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:11:33,116-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:14:35,967-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:14:36,292-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:14:36,607-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:14:37,811-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:14:40,053-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:18:30,356-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:18:30,711-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:18:30,951-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:18:32,138-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:18:32,140-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:20:25,453-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:20:25,885-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:20:26,175-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:20:27,702-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:20:28,119-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-27 19:20:57,216-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:20:57,696-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:20:57,963-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:20:59,423-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:20:59,793-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-27 19:21:42,092-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:21:42,596-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:21:43,002-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:21:44,701-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:21:45,314-->modify_excle.py modify_info 35-->INFO 客户ID为：273856
2021-10-27 19:21:45,976-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:21:46,019-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:21:49,216-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:24:05,241-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:24:05,669-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:24:05,950-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:24:07,324-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:24:07,700-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-27 19:25:18,429-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:25:18,858-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:25:19,118-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:25:20,606-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:25:25,153-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-27 19:25:52,662-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:25:53,112-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:25:53,378-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:25:55,028-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:25:56,667-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:25:56,709-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:26:01,963-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:26:34,111-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:26:34,557-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:26:34,989-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:26:36,535-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:26:37,582-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:26:37,833-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:26:43,036-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:36:58,649-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:36:59,030-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:36:59,393-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:37:00,824-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:37:05,277-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-27 19:42:42,237-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:42:42,666-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:42:43,056-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:42:44,468-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:42:45,183-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:42:45,921-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:42:50,345-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:45:12,368-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:45:12,767-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:45:13,015-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:45:14,421-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:45:15,094-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:45:15,130-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:47:27,413-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:47:27,847-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:47:28,128-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:47:29,629-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:47:34,365-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-27 19:47:58,417-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:47:58,834-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:47:59,102-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:48:00,542-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:51:03,456-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:51:03,935-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:51:04,197-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:51:05,597-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:51:06,289-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:51:06,336-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:51:10,684-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-27 19:51:42,799-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:51:43,262-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:51:43,543-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:51:44,924-->login.py base_message 40-->INFO 进入客户中心
2021-10-27 19:51:45,651-->import_case.py upload_file 26-->INFO 导入数据
2021-10-27 19:51:45,695-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-27 19:54:31,344-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-27 19:54:31,719-->login.py login_page 20-->INFO 登录成功
2021-10-27 19:54:32,009-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-27 19:54:33,124-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 10:08:30,089-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 10:08:30,595-->login.py login_page 20-->INFO 登录成功
2021-10-28 10:08:30,853-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 10:08:32,218-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 10:08:32,833-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 10:08:32,920-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 10:08:37,234-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-28 10:09:27,169-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002383F6F0D48>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/2f3cf9b9b11886efe3c59358104b1bb7/window
2021-10-28 10:09:29,219-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002384081B848>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/2f3cf9b9b11886efe3c59358104b1bb7/window
2021-10-28 10:09:31,254-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002384081BEC8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/2f3cf9b9b11886efe3c59358104b1bb7/window
2021-10-28 10:11:11,739-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 10:11:12,172-->login.py login_page 20-->INFO 登录成功
2021-10-28 10:11:12,576-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 10:11:14,499-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 10:11:15,336-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 10:11:15,495-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 10:11:24,758-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-28 10:14:03,735-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 10:14:04,231-->login.py login_page 20-->INFO 登录成功
2021-10-28 10:14:04,570-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 10:14:06,716-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 10:14:07,519-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 10:14:07,631-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 10:15:30,547-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 10:15:30,956-->login.py login_page 20-->INFO 登录成功
2021-10-28 10:15:31,230-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 10:15:32,523-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 10:36:03,163-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 11:10:49,305-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 11:10:50,685-->login.py login_page 20-->INFO 登录成功
2021-10-28 11:10:51,226-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 11:10:54,441-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 11:10:56,615-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 11:10:57,109-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 11:11:06,576-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-28 11:14:07,589-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 11:14:09,881-->login.py login_page 20-->INFO 登录成功
2021-10-28 11:14:10,443-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 11:14:13,772-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 11:14:16,486-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 11:14:16,612-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 11:14:26,181-->login_out.py out_login 21-->ERROR 'NoneType' object has no attribute 'click'
2021-10-28 14:31:42,464-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 14:31:43,085-->login.py login_page 20-->INFO 登录成功
2021-10-28 14:31:43,361-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 14:31:44,834-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 14:31:45,691-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 14:31:45,936-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 14:31:51,703-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 14:34:20,764-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 14:34:21,698-->login.py login_page 20-->INFO 登录成功
2021-10-28 14:34:22,126-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 14:34:24,583-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 14:34:25,668-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 14:34:25,740-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 14:34:31,056-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 14:35:51,442-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 14:35:52,385-->login.py login_page 20-->INFO 登录成功
2021-10-28 14:35:52,785-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 14:35:55,236-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 14:35:56,438-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 14:35:56,489-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 14:36:01,784-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 14:40:16,267-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 14:40:16,735-->login.py login_page 20-->INFO 登录成功
2021-10-28 14:40:17,016-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 14:40:18,614-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 14:40:19,245-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 14:40:19,289-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 14:40:24,499-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 14:43:58,792-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 14:43:59,266-->login.py login_page 20-->INFO 登录成功
2021-10-28 14:43:59,582-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 14:44:01,164-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 14:44:01,928-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 14:44:01,968-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 14:44:07,177-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 15:29:18,150-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 15:29:19,314-->login.py login_page 20-->INFO 登录成功
2021-10-28 15:29:19,582-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 15:29:22,364-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 15:29:23,625-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 15:29:24,106-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 15:29:29,476-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 17:42:45,954-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 17:42:46,357-->login.py login_page 20-->INFO 登录成功
2021-10-28 17:42:46,603-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 17:42:47,900-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 17:42:48,520-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 17:42:48,555-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 17:42:53,727-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 17:48:18,573-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 17:48:18,962-->login.py login_page 20-->INFO 登录成功
2021-10-28 17:48:19,234-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 17:48:20,533-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 17:48:21,114-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 17:48:21,154-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 17:49:26,114-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001696F614708>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/120ff95573ed893b012b68ad68e79eba/window
2021-10-28 17:49:28,148-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001696FD50E08>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/120ff95573ed893b012b68ad68e79eba/window
2021-10-28 17:49:30,183-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001696F7C6188>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/120ff95573ed893b012b68ad68e79eba/window
2021-10-28 17:50:29,431-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 17:50:29,842-->login.py login_page 20-->INFO 登录成功
2021-10-28 17:50:30,074-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 17:50:31,332-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 17:50:32,001-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 17:50:32,036-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 17:50:37,216-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 17:51:33,009-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 17:51:33,394-->login.py login_page 20-->INFO 登录成功
2021-10-28 17:51:33,649-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 17:51:34,925-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 17:51:35,556-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 17:51:35,597-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 17:51:40,793-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 18:54:06,249-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 18:54:06,650-->login.py login_page 20-->INFO 登录成功
2021-10-28 18:54:06,894-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 18:54:08,203-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 18:54:08,857-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 18:54:08,899-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 18:54:14,319-->login_out.py out_login 18-->INFO 退出登录成功
2021-10-28 19:09:54,346-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 19:09:54,715-->login.py login_page 20-->INFO 登录成功
2021-10-28 19:09:54,977-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 19:09:56,371-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 19:09:56,953-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 19:09:56,987-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 19:10:02,417-->login_out.py out_login 19-->INFO 退出登录成功
2021-10-28 19:15:14,462-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 19:15:14,852-->login.py login_page 20-->INFO 登录成功
2021-10-28 19:15:15,109-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-10-28 19:15:16,314-->login.py base_message 40-->INFO 进入客户中心
2021-10-28 19:15:16,938-->import_case.py upload_file 26-->INFO 导入数据
2021-10-28 19:15:16,978-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-10-28 19:15:17,391-->login_out.py out_login 19-->INFO 退出登录成功
2021-10-28 20:38:43,874-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 20:38:44,385-->login.py login_page 20-->INFO 登录成功
2021-10-28 20:40:36,949-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 20:40:37,431-->login.py login_page 20-->INFO 登录成功
2021-10-28 20:41:02,650-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 20:41:03,134-->login.py login_page 20-->INFO 登录成功
2021-10-28 20:41:44,316-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 20:41:44,791-->login.py login_page 20-->INFO 登录成功
2021-10-28 20:43:28,938-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 20:43:29,777-->login.py login_page 20-->INFO 登录成功
2021-10-28 21:01:27,563-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 21:01:27,969-->login.py login_page 20-->INFO 登录成功
2021-10-28 21:45:26,323-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-10-28 21:45:26,720-->login.py login_page 20-->INFO 登录成功
2021-11-01 10:26:18,705-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 10:26:19,208-->login.py login_page 20-->INFO 登录成功
2021-11-01 10:26:19,482-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 10:26:20,960-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 10:26:21,631-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 10:26:22,159-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 10:26:22,635-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 10:27:22,143-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 10:27:22,562-->login.py login_page 20-->INFO 登录成功
2021-11-01 10:27:22,816-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 10:27:24,096-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 10:27:24,752-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 10:27:24,790-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 10:27:25,201-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 10:28:13,078-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 10:28:13,630-->login.py login_page 20-->INFO 登录成功
2021-11-01 10:28:13,927-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 10:28:15,361-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 10:28:16,096-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 10:28:16,135-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 10:28:16,560-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 10:36:51,025-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 10:36:51,406-->login.py login_page 20-->INFO 登录成功
2021-11-01 10:36:51,684-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 10:36:53,100-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 10:36:53,687-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 10:36:53,725-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 10:36:54,142-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 10:38:08,014-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 10:38:08,405-->login.py login_page 20-->INFO 登录成功
2021-11-01 10:38:08,636-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 10:38:10,068-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 10:38:10,653-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 10:38:10,746-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 10:38:11,173-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 11:03:24,526-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:03:25,338-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:03:26,653-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 11:03:28,499-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 11:03:29,397-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 11:03:31,323-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 11:03:34,766-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: chrome=94.0.4606.61)

2021-11-01 11:03:34,766-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-01 11:06:51,168-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:06:51,692-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:06:52,782-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 11:06:54,170-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 11:06:54,731-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 11:06:57,833-->login_out.py out_login 23-->ERROR Message: element click intercepted: Element <a class="ant-dropdown-link user_link ant-dropdown-trigger" data-v-2468b04e="">...</a> is not clickable at point (1240, 32). Other element would receive the click: <div tabindex="-1" class="ant-modal-wrap " role="dialog" aria-labelledby="rcDialogTitle0">...</div>
  (Session info: chrome=94.0.4606.61)

2021-11-01 11:08:09,306-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:08:09,846-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:08:10,516-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 11:08:12,561-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 11:08:13,377-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 11:08:13,499-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 11:08:14,014-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 11:12:22,932-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:12:23,248-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:12:23,790-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 11:12:25,412-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 11:12:25,950-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 11:12:26,792-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 11:12:27,240-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 11:15:05,232-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:15:05,582-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:15:35,793-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:15:36,218-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:15:37,979-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 11:15:39,858-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 11:15:40,369-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 11:15:46,299-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 11:28:44,967-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:28:45,287-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:28:46,350-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 11:28:49,327-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 11:28:49,941-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 11:28:52,055-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 11:28:52,598-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 11:29:30,834-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:37:16,172-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 11:37:16,504-->login.py login_page 20-->INFO 登录成功
2021-11-01 11:37:17,597-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 11:37:19,536-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 11:37:20,155-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 11:37:20,590-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 11:37:23,915-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-01 11:37:23,915-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-01 14:32:03,499-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 14:32:03,831-->login.py login_page 20-->INFO 登录成功
2021-11-01 14:32:04,248-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 14:32:05,564-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 14:32:06,147-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 14:32:07,018-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 14:32:07,462-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 17:50:35,359-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 17:50:35,678-->login.py login_page 20-->INFO 登录成功
2021-11-01 17:50:35,937-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 17:50:37,118-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 17:50:37,724-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 17:50:37,840-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 17:50:38,286-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 17:55:09,130-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 17:55:09,420-->login.py login_page 20-->INFO 登录成功
2021-11-01 17:55:09,650-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 17:55:10,756-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 17:55:11,244-->import_case.py upload_file 26-->INFO 导入数据
2021-11-01 17:55:11,493-->import_case.py upload_file 29-->INFO 本次上传成功共:1条
2021-11-01 19:37:32,258-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 19:37:32,661-->login.py login_page 20-->INFO 登录成功
2021-11-01 19:37:32,911-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 19:37:34,590-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 19:37:35,251-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:37:35,333-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:41:01,523-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 19:41:02,069-->login.py login_page 20-->INFO 登录成功
2021-11-01 19:41:02,370-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 19:41:03,930-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 19:41:04,661-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:41:04,774-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:41:06,240-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:41:06,277-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:41:07,022-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 19:43:05,605-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 19:43:05,982-->login.py login_page 20-->INFO 登录成功
2021-11-01 19:43:06,227-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 19:43:07,657-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 19:43:08,333-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:43:08,396-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:43:09,006-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:43:09,049-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:43:11,162-->login_out.py out_login 23-->ERROR Message: element not interactable
  (Session info: headless chrome=94.0.4606.61)

2021-11-01 19:45:07,523-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 19:45:08,091-->login.py login_page 20-->INFO 登录成功
2021-11-01 19:45:08,617-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 19:45:10,384-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 19:45:11,210-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:45:11,255-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:45:11,888-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:45:11,913-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:45:13,332-->login_out.py out_login 23-->ERROR Message: element click intercepted: Element <a class="ant-dropdown-link user_link ant-dropdown-trigger" data-v-2468b04e="">...</a> is not clickable at point (1496, 32). Other element would receive the click: <div tabindex="-1" class="ant-modal-wrap " role="dialog" aria-labelledby="rcDialogTitle0" style="">...</div>
  (Session info: chrome=94.0.4606.61)

2021-11-01 19:47:50,653-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 19:47:51,511-->login.py login_page 20-->INFO 登录成功
2021-11-01 19:47:51,767-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 19:47:53,650-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 19:47:54,565-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:47:54,611-->import_case.py upload_file 33-->INFO 本次上传成功共:1条
2021-11-01 19:47:58,372-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 19:47:58,923-->import_case.py upload_file 33-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 19:48:03,076-->login_out.py out_login 23-->ERROR Message: element click intercepted: Element <a class="ant-dropdown-link user_link ant-dropdown-trigger" data-v-2468b04e="">...</a> is not clickable at point (1496, 32). Other element would receive the click: <div tabindex="-1" class="ant-modal-wrap " role="dialog" aria-labelledby="rcDialogTitle0" style="">...</div>
  (Session info: chrome=94.0.4606.61)

2021-11-01 20:06:37,018-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:06:38,173-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:06:39,034-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:06:40,963-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:06:42,038-->import_case.py upload_file 30-->INFO 导入数据
2021-11-01 20:06:42,082-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-01 20:06:46,418-->import_case.py upload_file 30-->INFO 导入数据
2021-11-01 20:06:47,109-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 20:06:51,312-->login_out.py out_login 23-->ERROR Message: element click intercepted: Element <a class="ant-dropdown-link user_link ant-dropdown-trigger" data-v-2468b04e="">...</a> is not clickable at point (1496, 32). Other element would receive the click: <div tabindex="-1" class="ant-modal-wrap " role="dialog" aria-labelledby="rcDialogTitle0" style="">...</div>
  (Session info: chrome=94.0.4606.61)

2021-11-01 20:23:19,010-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:23:20,358-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:23:20,946-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:23:23,030-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:23:23,780-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 20:23:25,845-->import_case.py upload_file 34-->INFO 123
2021-11-01 20:23:25,867-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-01 20:23:30,083-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 20:23:32,107-->import_case.py upload_file 34-->INFO 123
2021-11-01 20:23:32,131-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 20:23:36,255-->login_out.py out_login 23-->ERROR Message: element click intercepted: Element <a class="ant-dropdown-link user_link ant-dropdown-trigger" data-v-2468b04e="">...</a> is not clickable at point (1496, 32). Other element would receive the click: <div tabindex="-1" class="ant-modal-wrap " role="dialog" aria-labelledby="rcDialogTitle0" style="">...</div>
  (Session info: chrome=94.0.4606.61)

2021-11-01 20:25:37,282-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:25:37,837-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:25:38,299-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:25:40,113-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:25:40,844-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 20:26:18,209-->import_case.py upload_file 34-->INFO 123
2021-11-01 20:27:59,008-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 20:29:07,949-->import_case.py upload_file 34-->INFO 123
2021-11-01 20:31:10,640-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:31:11,274-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:31:11,757-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:31:13,542-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:31:14,310-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 20:31:14,470-->import_case.py upload_file 32-->INFO 取消导入
2021-11-01 20:31:14,504-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-01 20:31:18,232-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 20:31:18,649-->import_case.py upload_file 32-->INFO 取消导入
2021-11-01 20:31:18,686-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 20:31:22,520-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 20:35:00,738-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:35:01,416-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:35:01,821-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:35:03,688-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:35:04,464-->import_case.py upload_file 29-->INFO 导入数据
2021-11-01 20:35:59,773-->import_case.py upload_file 34-->INFO 定位失败
2021-11-01 20:51:35,311-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000225254F6B48>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/5b493f4bf12b571be375221f13c22d25/element
2021-11-01 20:51:37,780-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022525A67108>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/5b493f4bf12b571be375221f13c22d25/element
2021-11-01 20:51:39,815-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022525A67B08>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/5b493f4bf12b571be375221f13c22d25/element
2021-11-01 20:51:42,135-->get_element_file.py get_element 29-->INFO HTTPConnectionPool(host='127.0.0.1', port=62649): Max retries exceeded with url: /session/5b493f4bf12b571be375221f13c22d25/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000225258D3908>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2021-11-01 20:51:42,135-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-01 20:51:50,377-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:51:51,151-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:51:51,750-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:51:53,554-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:51:54,573-->import_case.py upload_file 34-->INFO 取消导入
2021-11-01 20:51:57,572-->import_case.py upload_file 34-->INFO 取消导入
2021-11-01 20:52:03,039-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: chrome=94.0.4606.61)

2021-11-01 20:52:03,040-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-01 20:53:06,165-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:53:06,864-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:53:07,180-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:53:09,023-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:53:10,040-->import_case.py upload_file 34-->INFO 取消导入
2021-11-01 20:53:10,087-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-01 20:53:14,807-->import_case.py upload_file 34-->INFO 取消导入
2021-11-01 20:53:14,850-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 20:53:18,530-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 20:55:15,005-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:55:15,608-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:55:15,939-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:55:17,636-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:55:50,749-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-01 20:56:22,443-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021B4BDD49C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/cb92ca9586db2b327fe2975155b64e7a/element
2021-11-01 20:56:24,761-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021B4C231EC8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/cb92ca9586db2b327fe2975155b64e7a/element
2021-11-01 20:56:27,069-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021B4BFBB1C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/cb92ca9586db2b327fe2975155b64e7a/element
2021-11-01 20:56:29,481-->get_element_file.py get_element 29-->INFO HTTPConnectionPool(host='127.0.0.1', port=55599): Max retries exceeded with url: /session/cb92ca9586db2b327fe2975155b64e7a/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021B4C25F248>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2021-11-01 20:56:29,481-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-01 20:56:31,516-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021B4BFB2D48>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/cb92ca9586db2b327fe2975155b64e7a/timeouts
2021-11-01 20:56:33,960-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021B4C25F748>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/cb92ca9586db2b327fe2975155b64e7a/timeouts
2021-11-01 20:56:36,250-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021B4C31C3C8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/cb92ca9586db2b327fe2975155b64e7a/timeouts
2021-11-01 20:56:44,583-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:56:46,278-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:56:46,673-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:56:49,284-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:56:54,436-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-01 20:56:54,495-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-01 20:57:00,496-->import_case.py upload_file 34-->INFO 取消导入
2021-11-01 20:57:00,660-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 20:57:04,911-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 20:59:00,195-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 20:59:02,634-->login.py login_page 20-->INFO 登录成功
2021-11-01 20:59:03,243-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 20:59:05,941-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 20:59:09,110-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-01 20:59:09,157-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-01 20:59:14,406-->import_case.py upload_file 34-->INFO 取消导入
2021-11-01 20:59:14,436-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 20:59:18,390-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-01 21:02:16,724-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-01 21:02:17,322-->login.py login_page 20-->INFO 登录成功
2021-11-01 21:02:17,737-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-01 21:02:20,052-->login.py base_message 40-->INFO 进入客户中心
2021-11-01 21:02:22,971-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-01 21:02:23,004-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-01 21:02:27,776-->import_case.py upload_file 34-->INFO 取消导入
2021-11-01 21:02:27,892-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-01 21:02:31,705-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-02 10:10:49,321-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 10:10:50,538-->login.py login_page 20-->INFO 登录成功
2021-11-02 10:10:51,179-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 10:10:53,125-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 10:10:57,012-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-02 10:10:57,169-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 10:11:02,443-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 10:11:02,459-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 10:11:08,413-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-02 11:31:43,166-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 11:31:44,054-->login.py login_page 20-->INFO 登录成功
2021-11-02 11:31:44,424-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 11:31:46,412-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 11:31:48,618-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-02 11:31:48,648-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:31:53,368-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:31:53,421-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 11:31:57,952-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:31:58,550-->import_case.py upload_file 38-->INFO 请按照模版上传数据,上传数据不能为空
2021-11-02 11:32:02,481-->login_out.py out_login 20-->INFO 退出登录成功
2021-11-02 11:40:11,265-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 11:40:11,734-->login.py login_page 20-->INFO 登录成功
2021-11-02 11:40:12,106-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 11:40:14,447-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 11:40:15,373-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:40:18,631-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:40:19,752-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:40:21,082-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:40:21,105-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:40:24,493-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 11:40:24,493-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 11:41:18,225-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 11:41:18,959-->login.py login_page 20-->INFO 登录成功
2021-11-02 11:41:19,730-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 11:41:22,215-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 11:41:23,584-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:41:25,098-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:41:26,961-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:41:27,001-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:41:29,713-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:41:29,796-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 11:41:33,261-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 11:41:33,262-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 11:43:17,458-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 11:43:18,234-->login.py login_page 20-->INFO 登录成功
2021-11-02 11:43:18,704-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 11:43:21,050-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 11:43:24,033-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-02 11:43:24,082-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:43:26,125-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:43:26,161-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 11:43:28,093-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:43:28,128-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 11:43:31,701-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 11:43:31,702-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 11:57:28,119-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 11:57:28,652-->login.py login_page 20-->INFO 登录成功
2021-11-02 11:57:29,098-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 11:57:31,147-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 11:57:32,314-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:57:33,191-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:57:35,274-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:57:35,441-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 11:57:37,081-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:57:37,210-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 11:57:40,697-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 11:57:40,697-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 11:58:21,001-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 11:58:21,809-->login.py login_page 20-->INFO 登录成功
2021-11-02 11:58:22,312-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 11:58:25,425-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 11:58:26,728-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:58:29,657-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:58:33,321-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 11:58:38,943-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 11:58:38,943-->login_out.py out_login 23-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 14:04:23,820-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 14:04:27,163-->login.py login_page 20-->INFO 登录成功
2021-11-02 14:04:28,031-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 14:04:33,459-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 14:04:36,470-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-02 14:04:36,611-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 14:04:41,417-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 14:04:42,058-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 14:04:44,480-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 14:04:44,973-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 14:04:46,295-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 17:47:36,624-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 17:47:37,055-->login.py login_page 20-->INFO 登录成功
2021-11-02 17:47:37,448-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 17:47:39,062-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 17:47:39,873-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 17:47:41,922-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 17:47:43,105-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 17:47:43,137-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 17:47:44,108-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 17:47:44,136-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 17:47:47,579-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 17:47:47,579-->login_out.py out_login 24-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 17:52:55,757-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 17:52:56,239-->login.py login_page 20-->INFO 登录成功
2021-11-02 17:52:56,660-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 17:52:58,624-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 17:52:59,592-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 17:53:00,847-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 17:53:02,129-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 17:53:02,174-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 17:53:03,879-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 17:53:03,907-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 17:53:04,555-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 18:00:59,601-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 18:01:00,052-->login.py login_page 20-->INFO 登录成功
2021-11-02 18:01:00,310-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 18:01:01,861-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 18:01:02,735-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 18:01:03,690-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 18:01:05,028-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 18:01:05,056-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 18:01:06,094-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 18:01:06,124-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 18:01:09,502-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 18:01:09,502-->login_out.py out_login 24-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 18:02:00,387-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 18:02:00,786-->login.py login_page 20-->INFO 登录成功
2021-11-02 18:02:01,387-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 18:02:03,026-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 18:02:03,947-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 18:02:04,013-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 18:02:05,066-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 18:02:05,089-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 18:02:06,458-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 18:02:06,481-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 18:02:09,879-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 18:02:09,880-->login_out.py out_login 24-->ERROR 'NoneType' object has no attribute 'click'
2021-11-02 18:03:32,352-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 18:03:33,561-->login.py login_page 20-->INFO 登录成功
2021-11-02 18:03:34,091-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 18:03:36,292-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 18:04:32,992-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 18:04:33,710-->login.py login_page 20-->INFO 登录成功
2021-11-02 18:04:34,401-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 18:04:36,073-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 18:05:31,188-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 18:05:33,380-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000185B3B38CC8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/25809fe5669108878274b6ae05cc707c/element
2021-11-02 18:05:35,656-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000185B3B81D88>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/25809fe5669108878274b6ae05cc707c/element
2021-11-02 19:03:54,440-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:03:54,959-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:03:55,313-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:03:56,997-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:03:57,900-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:03:57,949-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:03:58,489-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:03:58,516-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:03:59,334-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:03:59,362-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:04:00,336-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:09:03,178-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:09:03,864-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:09:04,343-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:09:06,498-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:09:08,606-->import_case.py upload_file 34-->INFO 成功导入数据
2021-11-02 19:09:08,651-->import_case.py upload_file 40-->INFO 本次上传成功共:1条
2021-11-02 19:09:10,077-->import_case.py upload_file 36-->INFO 取消导入
2021-11-02 19:09:10,106-->import_case.py upload_file 40-->INFO 本次上传成功共:1条
2021-11-02 19:09:11,571-->import_case.py upload_file 36-->INFO 取消导入
2021-11-02 19:09:11,600-->import_case.py upload_file 40-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:09:12,236-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:11:08,155-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:11:09,847-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:11:10,345-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:11:12,308-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:12:05,427-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:12:07,931-->connectionpool.py urlopen 781-->WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000027335044CC8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/007ea1ce9dce375a748f309842ee1539/timeouts
2021-11-02 19:12:24,734-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:12:26,413-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:12:26,876-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:12:29,145-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:13:49,949-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:15:39,593-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:16:33,054-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:16:33,096-->import_case.py upload_file 39-->INFO 请按照模版上传数据,上传数据不能为空
2021-11-02 19:16:50,630-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:18:24,698-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:18:25,559-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:18:25,979-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:18:27,798-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:18:29,904-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:18:29,944-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:18:31,490-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:18:31,520-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:18:32,990-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:18:33,041-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:18:33,659-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:21:26,239-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:21:26,860-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:21:27,828-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:21:30,014-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:21:31,218-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:21:32,083-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:21:38,492-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:21:42,684-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:21:42,707-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:21:48,131-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:23:39,489-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:23:40,140-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:23:40,607-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:23:42,420-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:23:51,330-->import_case.py upload_file 34-->INFO 成功导入数据
2021-11-02 19:24:01,040-->import_case.py upload_file 36-->INFO 取消导入
2021-11-02 19:24:10,080-->import_case.py upload_file 36-->INFO 取消导入
2021-11-02 19:24:13,516-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:25:44,467-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:25:44,948-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:25:45,416-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:25:47,141-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:25:48,244-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:25:48,325-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 19:25:49,962-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:25:49,993-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 19:25:51,385-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:25:51,413-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 19:25:52,021-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:27:14,860-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:27:15,521-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:27:16,031-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:27:17,941-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:27:18,834-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:27:18,873-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 19:27:25,359-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:27:25,390-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:27:31,539-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:27:35,065-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:29:29,516-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:29:30,045-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:29:30,597-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:29:32,404-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:29:33,332-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:29:33,363-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 19:29:39,859-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:29:39,895-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:29:46,086-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 19:29:47,201-->import_case.py upload_file 38-->INFO 上传失败
2021-11-02 19:29:52,865-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:31:48,928-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:31:49,442-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:31:50,188-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:31:52,509-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:31:57,028-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:32:01,937-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:32:01,995-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:32:08,387-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:32:11,808-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:35:16,948-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:35:17,549-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:35:18,245-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:35:19,982-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:35:25,787-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:35:32,223-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:35:32,317-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:35:40,364-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:35:40,396-->import_case.py upload_file 39-->INFO 上传失败
2021-11-02 19:35:45,894-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:45:19,833-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:45:20,356-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:45:21,516-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:45:23,958-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:45:29,033-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:45:32,112-->import_case.py upload_file 43-->INFO 没有定位到提示
2021-11-02 19:45:34,590-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:45:34,620-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:45:41,549-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:45:41,575-->import_case.py upload_file 39-->INFO 上传失败
2021-11-02 19:45:47,045-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:49:41,343-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:49:41,951-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:49:42,251-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:49:43,974-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:49:50,694-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:49:50,721-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:49:56,176-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:49:56,207-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:50:01,134-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:50:01,361-->import_case.py upload_file 39-->INFO 上传失败
2021-11-02 19:50:01,937-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 19:51:32,726-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 19:51:33,347-->login.py login_page 20-->INFO 登录成功
2021-11-02 19:51:33,667-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 19:51:35,400-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 19:51:40,858-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-02 19:51:40,885-->import_case.py upload_file 39-->INFO 本次上传成功共:1条
2021-11-02 19:51:45,406-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:51:45,451-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 19:51:49,762-->import_case.py upload_file 35-->INFO 取消导入
2021-11-02 19:51:49,794-->import_case.py upload_file 39-->INFO 上传失败
2021-11-02 19:51:50,420-->login_out.py out_login 21-->INFO 退出登录成功
2021-11-02 20:34:27,137-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-02 20:34:27,552-->login.py login_page 20-->INFO 登录成功
2021-11-02 20:34:27,801-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-02 20:34:29,409-->login.py base_message 40-->INFO 进入客户中心
2021-11-02 20:34:33,803-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 20:34:33,962-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-02 20:34:38,145-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 20:34:38,180-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-02 20:34:42,294-->import_case.py upload_file 34-->INFO 取消导入
2021-11-02 20:34:42,335-->import_case.py upload_file 38-->INFO 上传失败
2021-11-02 20:34:44,710-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-02 20:34:44,710-->login_out.py out_login 22-->ERROR 登录失败'NoneType' object has no attribute 'click'
2021-11-03 14:45:09,740-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 14:45:10,048-->login.py login_page 20-->INFO 登录成功
2021-11-03 14:45:10,257-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 14:45:11,169-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 14:45:14,734-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:45:17,785-->import_case.py upload_file 41-->INFO 没有定位到提示
2021-11-03 14:45:21,491-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:45:21,844-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 14:45:25,591-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:45:28,631-->import_case.py upload_file 41-->INFO 没有定位到提示
2021-11-03 14:45:30,790-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-03 14:45:30,790-->login_out.py out_login 22-->ERROR 登录失败'NoneType' object has no attribute 'click'
2021-11-03 14:47:05,075-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 14:47:05,331-->login.py login_page 20-->INFO 登录成功
2021-11-03 14:47:05,511-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 14:47:06,421-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 14:47:11,332-->import_case.py upload_file 32-->INFO 成功导入数据
2021-11-03 14:47:11,350-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-03 14:47:15,221-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:47:15,252-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 14:47:19,125-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:47:19,149-->import_case.py upload_file 38-->INFO 上传失败
2021-11-03 14:47:21,546-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-03 14:47:21,547-->login_out.py out_login 22-->ERROR 登录失败'NoneType' object has no attribute 'click'
2021-11-03 14:48:51,266-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 14:48:51,832-->login.py login_page 20-->INFO 登录成功
2021-11-03 14:48:52,128-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 14:48:53,786-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 14:48:58,123-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:48:58,156-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-03 14:49:02,225-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:49:02,256-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 14:49:06,189-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:49:06,214-->import_case.py upload_file 38-->INFO 上传失败
2021-11-03 14:49:06,750-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 14:52:26,384-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 14:52:26,743-->login.py login_page 20-->INFO 登录成功
2021-11-03 14:52:26,978-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 14:52:28,120-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 14:52:32,111-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:52:32,380-->import_case.py upload_file 38-->INFO 本次上传成功共:1条
2021-11-03 14:52:36,189-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:52:36,218-->import_case.py upload_file 38-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 14:52:39,821-->import_case.py upload_file 34-->INFO 取消导入
2021-11-03 14:52:39,844-->import_case.py upload_file 38-->INFO 上传失败
2021-11-03 14:52:40,162-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 14:53:24,085-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 14:53:24,392-->login.py login_page 20-->INFO 登录成功
2021-11-03 14:53:24,620-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 14:53:25,801-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 14:53:33,292-->import_case.py upload_file 33-->INFO 成功导入数据
2021-11-03 14:53:36,338-->import_case.py upload_file 42-->INFO 没有定位到提示
2021-11-03 14:53:41,009-->import_case.py upload_file 35-->INFO 取消导入
2021-11-03 14:53:41,027-->import_case.py upload_file 39-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 14:53:45,613-->import_case.py upload_file 35-->INFO 取消导入
2021-11-03 14:53:45,624-->import_case.py upload_file 39-->INFO 上传失败
2021-11-03 14:53:46,079-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 14:58:09,194-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 14:58:09,528-->login.py login_page 20-->INFO 登录成功
2021-11-03 14:58:09,820-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 14:58:10,955-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 14:58:14,420-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 14:58:14,531-->import_case.py upload_file 41-->INFO 取消导入
2021-11-03 14:58:18,247-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 14:58:18,443-->import_case.py upload_file 41-->INFO 取消导入
2021-11-03 14:58:21,896-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 14:58:22,039-->import_case.py upload_file 41-->INFO 取消导入
2021-11-03 14:58:22,532-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 14:59:17,495-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 14:59:17,846-->login.py login_page 20-->INFO 登录成功
2021-11-03 14:59:18,066-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 14:59:19,170-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 14:59:23,545-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 14:59:27,608-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 14:59:31,072-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 14:59:32,207-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 14:59:38,622-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-03 14:59:39,763-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 14:59:40,227-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 15:00:24,603-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 15:00:24,960-->login.py login_page 20-->INFO 登录成功
2021-11-03 15:00:25,185-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 15:00:26,349-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 15:00:30,241-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 15:00:34,281-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 15:00:37,762-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 15:00:38,905-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 15:00:42,966-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 15:00:44,101-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 15:00:44,549-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 15:15:08,517-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 15:15:08,743-->login.py login_page 20-->INFO 登录成功
2021-11-03 15:15:08,929-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 15:15:09,881-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 15:15:13,465-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 15:15:17,513-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 15:15:21,042-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 15:15:22,194-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 15:15:25,677-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 15:15:26,810-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 15:15:29,145-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-03 15:15:29,145-->login_out.py out_login 22-->ERROR 登录失败'NoneType' object has no attribute 'click'
2021-11-03 15:18:51,960-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 15:18:52,209-->login.py login_page 20-->INFO 登录成功
2021-11-03 15:18:52,403-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 15:18:53,331-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 15:18:57,002-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 15:19:01,068-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 15:19:04,617-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 15:19:05,798-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 15:19:09,379-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 15:19:10,583-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 15:19:12,815-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 19:08:05,616-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 19:08:05,778-->login.py login_page 20-->INFO 登录成功
2021-11-03 19:08:05,945-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 19:08:06,544-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 19:08:10,446-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 19:08:14,486-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 19:08:18,248-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 19:08:19,393-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:08:22,776-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 19:08:23,884-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:08:26,108-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 19:42:50,045-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 19:42:50,213-->login.py login_page 20-->INFO 登录成功
2021-11-03 19:42:50,380-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 19:42:50,980-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 19:42:54,422-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 19:42:58,461-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 19:43:01,901-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 19:43:03,023-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:43:09,360-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-03 19:43:10,471-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:43:15,585-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-03 19:43:15,585-->login_out.py out_login 22-->ERROR 退出登录失败'NoneType' object has no attribute 'click'
2021-11-03 19:48:33,036-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 19:48:33,211-->login.py login_page 20-->INFO 登录成功
2021-11-03 19:48:33,379-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 19:48:33,968-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 19:48:37,463-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 19:48:41,493-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 19:48:44,938-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 19:48:46,066-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:48:49,435-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 19:48:50,554-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:48:52,765-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 19:48:55,219-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 19:48:55,382-->login.py login_page 20-->INFO 登录成功
2021-11-03 19:48:55,552-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 19:48:56,134-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 19:48:59,593-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 19:49:03,633-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 19:49:07,069-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 19:49:08,193-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:49:11,545-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 19:49:12,660-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:49:14,854-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 19:55:58,564-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 19:55:58,728-->login.py login_page 20-->INFO 登录成功
2021-11-03 19:55:58,893-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 19:55:59,420-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 19:56:02,861-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 19:56:06,948-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 19:56:10,367-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 19:56:11,490-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:56:14,939-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 19:56:16,044-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:56:18,232-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 19:56:20,611-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 19:56:20,770-->login.py login_page 20-->INFO 登录成功
2021-11-03 19:56:20,974-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 19:56:21,510-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 19:56:24,943-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 19:56:28,988-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 19:56:32,451-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 19:56:33,581-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:56:36,952-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 19:56:38,096-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 19:56:40,298-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 20:00:59,070-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:00:59,239-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:00:59,397-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:00:59,951-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:01:03,380-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:01:07,443-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:01:10,859-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 20:01:11,978-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:01:15,326-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 20:01:16,449-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:01:18,647-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 20:01:20,968-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:01:21,125-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:01:21,286-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:01:21,841-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:01:25,305-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:01:29,357-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:01:32,772-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 20:01:33,889-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:01:37,270-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 20:01:38,388-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:01:40,600-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 20:16:03,225-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:16:18,904-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:16:19,064-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:16:19,329-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:16:20,090-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:16:23,586-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:16:27,645-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:16:31,256-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 20:16:32,382-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:16:38,707-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-03 20:16:39,840-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:16:44,966-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-03 20:16:44,966-->login_out.py out_login 22-->ERROR 退出登录失败'NoneType' object has no attribute 'click'
2021-11-03 20:17:47,422-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:17:47,597-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:17:47,959-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:17:48,694-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:17:52,261-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:17:56,326-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:18:02,623-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-03 20:18:03,760-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:18:10,178-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-03 20:18:11,290-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:18:13,467-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 20:18:24,531-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:18:24,699-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:18:25,049-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:18:26,042-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:18:29,825-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:18:33,921-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:18:40,251-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-03 20:18:41,376-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:18:47,598-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-03 20:18:48,711-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:18:50,956-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 20:20:02,930-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:20:03,096-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:20:05,684-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:20:32,074-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"css selector","selector":"#app > section > header > div.user > a"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-03 20:20:32,075-->login_out.py out_login 22-->ERROR 退出登录失败'NoneType' object has no attribute 'click'
2021-11-03 20:20:36,753-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:20:36,922-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:20:37,255-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:20:37,856-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:20:41,864-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:20:45,920-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:20:49,599-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 20:20:50,725-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:20:54,261-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 20:20:55,385-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:20:57,596-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 20:21:52,566-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:21:52,756-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:21:52,930-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:21:53,508-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:21:56,998-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:22:01,054-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:22:04,504-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 20:22:05,637-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:22:09,016-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 20:22:10,137-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:22:12,348-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-03 20:22:14,698-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-03 20:22:14,885-->login.py login_page 20-->INFO 登录成功
2021-11-03 20:22:15,066-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-03 20:22:15,638-->login.py base_message 40-->INFO 进入客户中心
2021-11-03 20:22:19,117-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-03 20:22:23,180-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-03 20:22:26,853-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-03 20:22:28,013-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:22:32,157-->import_case.py upload_file 31-->INFO 上传失败
2021-11-03 20:22:33,269-->import_case.py upload_file 42-->INFO 取消导入
2021-11-03 20:22:35,488-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-04 10:09:13,939-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-04 10:09:14,151-->login.py login_page 20-->INFO 登录成功
2021-11-04 10:09:14,320-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-04 10:09:14,930-->login.py base_message 40-->INFO 进入客户中心
2021-11-04 10:09:18,319-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-04 10:09:22,397-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-04 10:09:27,103-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-04 10:09:28,246-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:09:31,778-->import_case.py upload_file 31-->INFO 上传失败
2021-11-04 10:09:32,906-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:09:35,129-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-04 10:09:37,500-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-04 10:09:37,710-->login.py login_page 20-->INFO 登录成功
2021-11-04 10:09:37,938-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-04 10:09:38,535-->login.py base_message 40-->INFO 进入客户中心
2021-11-04 10:09:42,053-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-04 10:09:46,115-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-04 10:09:49,610-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-04 10:09:50,761-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:09:54,178-->import_case.py upload_file 31-->INFO 上传失败
2021-11-04 10:09:55,303-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:09:57,503-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-04 10:11:12,845-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-04 10:11:13,031-->login.py login_page 20-->INFO 登录成功
2021-11-04 10:11:13,196-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-04 10:11:13,872-->login.py base_message 40-->INFO 进入客户中心
2021-11-04 10:11:18,594-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-04 10:11:22,648-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-04 10:11:26,262-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-04 10:11:27,406-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:11:30,846-->import_case.py upload_file 31-->INFO 上传失败
2021-11-04 10:11:31,976-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:11:34,192-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-04 10:11:36,658-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-04 10:11:36,829-->login.py login_page 20-->INFO 登录成功
2021-11-04 10:11:37,164-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-04 10:11:37,755-->login.py base_message 40-->INFO 进入客户中心
2021-11-04 10:11:41,226-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-04 10:11:45,295-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-04 10:11:49,703-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-04 10:11:50,851-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:11:54,291-->import_case.py upload_file 31-->INFO 上传失败
2021-11-04 10:11:55,440-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 10:11:57,646-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-04 15:10:01,155-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-04 15:10:01,330-->login.py login_page 20-->INFO 登录成功
2021-11-04 15:10:01,492-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-04 15:10:02,055-->login.py base_message 40-->INFO 进入客户中心
2021-11-04 15:10:05,495-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-04 15:10:09,568-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-04 15:10:13,044-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-04 15:10:14,161-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 15:10:17,510-->import_case.py upload_file 31-->INFO 上传失败
2021-11-04 15:10:18,615-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 15:10:20,791-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-04 15:10:23,211-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-04 15:10:23,372-->login.py login_page 20-->INFO 登录成功
2021-11-04 15:10:23,534-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-04 15:10:24,127-->login.py base_message 40-->INFO 进入客户中心
2021-11-04 15:10:27,671-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-04 15:10:31,744-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-04 15:10:35,192-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-04 15:10:36,314-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 15:10:39,721-->import_case.py upload_file 31-->INFO 上传失败
2021-11-04 15:10:40,872-->import_case.py upload_file 42-->INFO 取消导入
2021-11-04 15:10:43,066-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-05 11:38:09,495-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-05 11:38:09,667-->login.py login_page 20-->INFO 登录成功
2021-11-05 11:38:10,337-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-05 11:38:10,951-->login.py base_message 40-->INFO 进入客户中心
2021-11-05 11:38:14,769-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-05 11:38:18,807-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-05 11:38:22,686-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-05 11:38:23,812-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 11:38:27,262-->import_case.py upload_file 31-->INFO 上传失败
2021-11-05 11:38:28,372-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 11:38:30,569-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-05 11:38:42,462-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-05 11:38:42,632-->login.py login_page 20-->INFO 登录成功
2021-11-05 11:38:43,278-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-05 11:38:43,918-->login.py base_message 40-->INFO 进入客户中心
2021-11-05 11:38:47,381-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-05 11:38:51,429-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-05 11:38:54,819-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-05 11:38:55,937-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 11:38:59,317-->import_case.py upload_file 31-->INFO 上传失败
2021-11-05 11:39:00,432-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 11:39:02,605-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-05 17:19:09,476-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-05 17:19:09,753-->login.py login_page 20-->INFO 登录成功
2021-11-05 17:19:09,986-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-05 17:19:10,998-->login.py base_message 40-->INFO 进入客户中心
2021-11-05 17:19:15,116-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-05 17:19:19,144-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-05 17:19:22,714-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-05 17:19:23,890-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 17:19:30,340-->import_case.py upload_file 33-->INFO 没有定位到提示
2021-11-05 17:19:31,495-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 17:19:36,655-->get_element_file.py get_element 29-->INFO Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[4]/div/div/div/span[5]"}
  (Session info: headless chrome=94.0.4606.61)

2021-11-05 17:19:36,655-->login_out.py out_login 22-->ERROR 退出登录失败'NoneType' object has no attribute 'click'
2021-11-05 17:19:39,316-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-05 17:19:39,608-->login.py login_page 20-->INFO 登录成功
2021-11-05 17:19:39,809-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-05 17:19:40,878-->login.py base_message 40-->INFO 进入客户中心
2021-11-05 17:19:44,625-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-05 17:19:48,672-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-05 17:19:52,234-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-05 17:19:53,440-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 17:19:57,027-->import_case.py upload_file 31-->INFO 上传失败
2021-11-05 17:19:58,244-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 17:20:00,566-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-05 19:29:17,121-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-05 19:29:17,291-->login.py login_page 20-->INFO 登录成功
2021-11-05 19:29:17,470-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-05 19:29:18,041-->login.py base_message 40-->INFO 进入客户中心
2021-11-05 19:29:21,502-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-05 19:29:25,566-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-05 19:29:29,038-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-05 19:29:30,168-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 19:29:33,533-->import_case.py upload_file 31-->INFO 上传失败
2021-11-05 19:29:34,645-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 19:29:36,837-->login_out.py out_login 19-->INFO 退出登录成功
2021-11-05 19:29:47,863-->driver.py open_driver 40-->INFO 浏览器启动成功
2021-11-05 19:29:48,039-->login.py login_page 20-->INFO 登录成功
2021-11-05 19:29:48,212-->choose_space.py choose_space 20-->INFO 进入空间：测试复制FS空间
2021-11-05 19:29:48,764-->login.py base_message 40-->INFO 进入客户中心
2021-11-05 19:29:52,233-->import_case.py upload_file 31-->INFO 本次上传成功共:1条
2021-11-05 19:29:56,276-->import_case.py upload_file 40-->INFO 成功导入数据
2021-11-05 19:29:59,900-->import_case.py upload_file 31-->INFO 一次只允许上传最多10个批次号数据
2021-11-05 19:30:01,019-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 19:30:04,368-->import_case.py upload_file 31-->INFO 上传失败
2021-11-05 19:30:05,475-->import_case.py upload_file 42-->INFO 取消导入
2021-11-05 19:30:07,667-->login_out.py out_login 19-->INFO 退出登录成功
